### ISSUE TITLE
Fails if no GitHub/GitLab token and release disabled

### ISSUE THREAD

**Issue by fheidenreich** [goreleaser/goreleaser#1073]
Since the introduction of GitLab release in #1038 either a GitHub or GitLab token is required for the build to succeed even if releases are disabled via

```
release:
  disable: true
```

The error message is
```
release failed after 6.92s error=GitHub/GitLab Releases: failed to publish artifacts: client is not yet implemented:
```

**Comment by caarlos0** [comment]
thanks!

**Comment by izgeri** [comment]
@fheidenreich I am hitting this as well - it looks like the fix will be included in the 1.0 release, is there a target date for when that will be made available?

**Comment by caarlos0** [comment]
will do it now :)
